README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
Hey guys,

A few points to add here:
* In #63, I have moved the `branch_name` attribute to a separate model that's just for pull requests. Pull requests are always based around branches, so there's no need to rename the attribute. What's more, I've identified [a simple TODO cleanup](https://github.com/open-craft/opencraft/pull/63/files#diff-4ccb9a3bd803e3e653eac6d28f3d5be4R103) that will eliminate the need to call any GitHub API to get the "tip commit" for a given ref, and at the same time fix the bug where OpenCraft IM breaks when a PR branch is deleted. (This is done by simple getting the sha1 hash directly from the PR details when querying the API for PRs, rather than making a separate call)
* After #63, instances and appservers have an `edx_platform_version` attribute, which can be any type of git ref - branch, commit, tag, etc. all work perfectly fine.
* The only catch is that the `named-release/dogwood` style tags are only on the `edx/edx-platform` fork, and you will get errors if you try to use those tags with the `open-craft` fork.